odoo/tools/float_utils.py

```diff
@@ -76,10 +101,10 @@ def float_round(value, precision_digits=None, precision_rounding=None, rounding_
     # In order to easily support rounding to arbitrary 'steps' (e.g. coin values),
     # we normalize the value before rounding it as an integer, and de-normalize
     # after rounding: e.g. float_round(1.3, precision_rounding=.5) == 1.5
-    def normalize(val):
+    def normalize(val: float) -> float:
         return val / rounding_factor
 
-    def denormalize(val):
+    def denormalize(val: float) -> float:
```

Comment:
Typing internal functions is not very useful, unless the signature is nontrivial.  Please remove those types.

Reply:
Done!